src/performance_profiles.jl

```diff
@@ -26,6 +26,17 @@ function performance_ratios(T :: Array{Float64,2}; logscale :: Bool=true)
   for p = 1 : np
     r[p, :] = T[p, :] / minperf[p];
   end
+  # Use a draw tolerance of `drawtol` (in percentage).
+  if (drawtol > 0. && drawtol < 1.)
+    drawtol += 1. 
+    for p in 1 : np
+      for s in 1 : ns
+        if r[p,s] <= drawtol
+          r[p,s] = 1.
+        end
+      end
+    end
```

Comment:
I think this can be simplified to
```suggestion
    r[r .<= drawtol] .= 1
```

Reply:
Great!